### PR TITLE
[rllib] different activation support

### PR DESCRIPTION
**Pull request by Scitator**
Hi,

For DDPG (and other RL algorithms) it's common to try different activation functions. So, here is a DDPG support pull request.

### CODE REVIEW

File: python/ray/rllib/models/visionnet.py [hunk 1]
```diff
@@ -17,10 +17,14 @@ def _init(self, inputs, num_outputs, options):
             [32, [4, 4], 2],
             [512, [10, 10], 1],
         ])
+        conv_activation = options.get("conv_activation", "relu")
+        activation = tf.nn.__dict__[conv_activation]
```
Comment (richardliaw) [inline python/ray/rllib/models/visionnet.py]: oh nice; is this standard?

Comment (Scitator) [reply python/ray/rllib/models/visionnet.py]: it python :)
I found this solution quite powerful and easy to go

Comment (Scitator) [reply python/ray/rllib/models/visionnet.py]: by this way, it's possible to use any tf activation functions tanh/elu/whatever without any performance troubles

File: python/ray/rllib/ddpg/ddpg_policy_graph.py [hunk 2]
```diff
@@ -26,11 +26,12 @@ class PNetwork(object):
     """Maps an observations (i.e., state) to an action where each entry takes
     value from (0, 1) due to the sigmoid function."""
 
-    def __init__(self, model, dim_actions, hiddens=[64, 64]):
+    def __init__(self, model, dim_actions, hiddens=[64, 64], activation='relu'):
```
Comment (ericl) [inline python/ray/rllib/ddpg/ddpg_policy_graph.py]: nit: we prefer double quotes when possible

File: python/ray/rllib/models/visionnet.py [hunk 3]
```diff
@@ -17,10 +17,14 @@ def _init(self, inputs, num_outputs, options):
             [32, [4, 4], 2],
             [512, [10, 10], 1],
         ])
+        conv_activation = options.get("conv_activation", "relu")
```
Comment (ericl) [inline python/ray/rllib/models/visionnet.py]: Could you also add this to the list of options in catalog.py?

**Review by ericl — COMMENTED** [review]
Looks good, a few minor comments

**Review by ericl — APPROVED** [review]
Looks good, will merge after test runs

**Comment by ericl** [comment]
Looks like there are some conflicts.

**Comment by richardliaw** [comment]
Test failures seem unrelated (jenkins is only failing on post-rllib tests)

**Comment by richardliaw** [comment]
@Scitator, thanks for contributing this!

**Pull request merged**